Sources/Classes/core/Display/RealMiniAppView.swift

```diff
@@ -242,6 +257,7 @@ internal class RealMiniAppView: UIView {
         UIViewController.attemptRotationToDeviceOrientation()
         webView.configuration.userContentController.removeMessageHandler()
         NotificationCenter.default.removeObserver(self)
+        secureStorage.unloadStorage()
```

Comment:
We have to make sure we test this thing that we don't have any memory leak.

Reply:
@rleojoseph deinit is called for both I'll add a deinit logging for MiniAppSecureStorage. I think for more proper Unit Tests we need a byte more time.